.project

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>Opus</name>
-	<comment>Project Opus created by Buildship.</comment>
+	<name>main</name>
+	<comment>Project main created by Buildship.</comment>
```

Comment:
sorry always have this problem when i reinstall eclipse. anyone has any idea how to untrack it?